crates/wit/src/commands/add.rs

```diff
@@ -127,16 +123,6 @@ impl AddCommand {
                     .dependencies
                     .insert(id.clone(), Dependency::Package(package));
 
-                // Resolve all dependencies to ensure that the lockfile is up to date.
-                resolve_dependencies(
```

Comment:
I am sure i am missing something, but why is this removed?

Reply:
This was to fix a double-resolution that was taking place, once to resolve the package being added and then again to resolve all packages in an attempt to update the lock file.

Updating the lock file isn't strictly necessary from the command, so this was removed to improve the UX.

Reply:
Awesome, makes complete sense. Thanks for the explanation.